### PR TITLE
(IAC-303) Stringify module

### PR DESCRIPTION
**Pull request by sheenaajay**

### CODE REVIEW

File: REFERENCE.md [hunk 1]
```diff
@@ -5,21 +5,26 @@
...
 **Defined types**
 
-* [`helm::chart`](#helmchart): == helm::chart
-* [`helm::chart_update`](#helmchart_update): 
-* [`helm::create`](#helmcreate): == helm::create
-* [`helm::helm_init`](#helmhelm_init): == helm::helm_init
-* [`helm::package`](#helmpackage): == helm::package
-* [`helm::repo`](#helmrepo): == helm::repo
-* [`helm::repo_update`](#helmrepo_update): == helm::repo_update
+* [`helm::chart`](#helmchart): Creates a new Helm chart.
```
Comment (carabasdaniel) [inline REFERENCE.md]: manages the deployment of helm charts

File: REFERENCE.md [hunk 2]
```diff
@@ -5,21 +5,26 @@
...
+* [`helm::chart_update`](#helmchart_update): A module to update Helm charts.
+* [`helm::create`](#helmcreate): A module to create Helm charts.
```
Comment (carabasdaniel) [inline REFERENCE.md]: Creates a new Helm chart

File: REFERENCE.md [hunk 3]
```diff
@@ -5,21 +5,26 @@
...
 
-* [`helm::chart`](#helmchart): == helm::chart
-* [`helm::chart_update`](#helmchart_update): 
-* [`helm::create`](#helmcreate): == helm::create
-* [`helm::helm_init`](#helmhelm_init): == helm::helm_init
-* [`helm::package`](#helmpackage): == helm::package
-* [`helm::repo`](#helmrepo): == helm::repo
-* [`helm::repo_update`](#helmrepo_update): == helm::repo_update
+* [`helm::chart`](#helmchart): Creates a new Helm chart.
+* [`helm::chart_update`](#helmchart_update): A module to update Helm charts.
```
Comment (carabasdaniel) [inline REFERENCE.md]: Update the deployed Helm charts

**Comment by sheenaajay** [comment]
@carabasdaniel Thank you. Incorporated the review comments.

**Comment by carabasdaniel** [comment]
👍 LGTM

**Pull request merged**